import_upstream.bash

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export PYTHONPATH=/home/rosbuild/reprepro_updater/src
```

Comment:
Should we be hard coding this? This is a public repository, I think it shouldn't contain the folder structure of our server if possible.